fe/fe-core/src/main/java/com/starrocks/sql/ast/Property.java

```diff
@@ -0,0 +1,22 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.ast;
+
+import com.starrocks.analysis.ParseNode;
+
+public class Property implements ParseNode {
```

Comment:
`Property` is a trivial name...

Reply:
property corresponds to the property in the grammar, I think it is no problem to directly property here, and this class is in the AST package, so it will not cause confusion caused by naming